### DISCUSSION TITLE
PCA DAN KMEANS

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#293] [Q&A] [answered]
permisi izin bertanya terkait unsupervised learning mengenai Kmeans cluster dan pca itu apakah hal yang berbeda ya?apakah mereka bisa dintegrasikan satu sama lain antara pca dan cluster

**Comment by NabiilahArdini** [comment, marked as answer]
Halo Pak @daryraa ,
untuk PCA dan Clustering memang berbeda tujuan. PCA untuk reduksi dimensi/kolom data (serta hasilnya berupa kolom2 baru (PC) yang memang tidak bisa diinterpretasikan). Clustering untuk mengelompokkan data (serta hasilnya diharapkan dapat dilanjutkan untuk profiling/karakterisasi/interpretasi cluster). 2 hal tersebut berbeda secara maksud utamanya dan jarang diintegrasikan. 

Namun, PCA dan Clustering dapat diintegrasikan di sisi yang lain, yaitu penggunaan Biplot dari PCA untuk visualisasi high dimensional data (plot sebaran data) untuk cek outlier dari data. Ini ada dibagian EDA sebelum clustering. Bila ada outlier maka bisa dibuang, karena hadirnya outlier dapat mengganggu proses clustering.

Semoga informasi ini dapat bermanfaat!
